### ISSUE TITLE
Big Pharma Handing Hospitals The Cash

### ISSUE THREAD

**Issue by barjacks** [jsoma/playfair-projects#199]
# Story:

Medical professionals receiving donations from pharmaceutical companies has been a conflicting issue for a long time. For the first time this summer 57 pharmaceutical companies published the figures. So, also for the first time, this analysis shows which hospitals gets the most cash from big pharma.
# Datasource:

Here's the data, cleaned for the [top 7 hospitals](https://docs.google.com/spreadsheets/d/1uq7S2QMEVlLnfKuO3x9NxyNKBf4T6D0uwEMDZXg0QC8/edit?usp=sharing), (but not for the rest of them.)
# Summary:

Digging deeper the analysis shows, which hospitals the large companies have homed in on. Novartis, e.g., dominates the Kantonsspital Basel, whereas Bayer seem pretty fixated ob the Berner Inselspital.
# Headlines:
- Big Pharma's Favorite Hospitals - at Least When it Comes to Handing Over the Cash
# Graphic:

![hospitals_eng](https://cloud.githubusercontent.com/assets/1587242/17541530/86f1180e-5e8e-11e6-8a49-8136008a3c14.png)
# Story issue checklist

My pitch was (use the number): #161 
- [x] My pitch has been approved (see `PITCHING.md`)
- [x] My story issue links to my pitch issue
- [x] I link to my finalized (ish) data source(s)
- [x] I've included a brief summary of my story
- [x] I've included some possible headlines or findings
- [ ] I've included some links or images as inspiration (if you have any)
- [ ] I have received two comments of peer feedback
- [x] I've included an update of my visualization/story in a comment
- [x] I have received two comments of peer feedback after posting an update
- [ ] I have received editorial feedback

**Comment by spm2164** [comment]
Interesting story and I love your headline, but the layout here could def use some work. Lots of white space - I'd either cut something down or add something more descriptive. Also, I'd love to see this hospital just rolling around in cash vs this small hospital that none of these companies seem to care much about.

**Comment by palrogg** [comment]
Great graphs. Just two remarks about the layout:
* I love the “Distribution by Canton” part. Maybe could it be a little more highlighted?
* I'm also not sure about the pale green color, even if it looks like an hospital wall (for the 7 hospitals which get the most).

**Comment by thisss** [comment]
There's a typo: You accidentally labeled Bern as sleepy. It rather should be called "dynamic". ;-) Great job. I would change "Universitätsspital Bern (Insel)" to "Inselspital Bern (Universitätsspital)".

**Comment by barjacks** [comment]
I've played around with the colors a bit more. And also with an image, to show the hospital: But I just can't make it look nice. So I took it out again. I introduced two more graphs though. One showing which hospitals gets the most donations and the ones with the highest travel expenses (doctors from Zurich's main hospital apparently like to travel quite a bit more than doctors from other hospitals).

PS: I can't find that typo, though @thisss 
![hospitals_eng_new](https://cloud.githubusercontent.com/assets/1587242/17602625/75e8b3fc-5fdb-11e6-95eb-ec6d9e5db320.png)



![hospitals_eng_new](https://cloud.githubusercontent.com/assets/1587242/17602527/01de0610-5fdb-11e6-9504-fb1818a4796e.png)
